projects/dsp-ui/src/lib/viewer/values/link-value/link-value.component.spec.ts

```diff
@@ -407,6 +407,15 @@ describe('LinkValueComponent', () => {
 
     }));
 
+    it('should emit the displayValue when the value is clicked on', () => {
+        spyOn(testHostComponent.inputValueComponent, 'refResClicked').and.callThrough();
```

Comment:
I only use spies on code that I mock (dependencies of the code to be tested) since I am not sure how spies change the behaviour of the code I want to test.

Instead of using a spy, you could just produce a side-effect when the `@Output` is fired:

https://github.com/dasch-swiss/dsp-ui-lib/blob/6ee18e416ffda6691185fad052db49dc8b9e7eef/projects/dsp-ui/src/lib/search/advanced-search/select-ontology/select-ontology.component.spec.ts#L14-L44

The you can just check if the emitted value has been assigned:


https://github.com/dasch-swiss/dsp-ui-lib/blob/6ee18e416ffda6691185fad052db49dc8b9e7eef/projects/dsp-ui/src/lib/search/advanced-search/select-ontology/select-ontology.component.spec.ts#L117-L133

Reply:
fixed in 9064a60